### ISSUE TITLE
Frequently asked question example to connect to an existing application doesn't work.

### ISSUE THREAD

**Issue by johndlawlor** [microsoft/WinAppDriver#1156]
I am following these instructions to connect to an existing instance of notepad.
https://github.com/Microsoft/WinAppDriver/wiki/Frequently-Asked-Questions/a8c02cfac47b4bf0c12c571b6010c403dcfe5e7f#when-and-how-to-attach-to-an-existing-app-window

But they don't seem to work as they always launch a new notepad instance. 

        private const string NotepadAppId = @"C:\Windows\System32\notepad.exe";
        protected static WindowsDriver<WindowsElement> session;

        public static void SetupExisting(TestContext context)
        {
            DesiredCapabilities appCapabilities = new DesiredCapabilities();
            appCapabilities.SetCapability("app", NotepadAppId);
            session = new WindowsDriver<WindowsElement>(new Uri(WindowsApplicationDriverUrl), appCapabilities);
            Assert.IsNotNull(session);

            var currentWindowHandle = session.CurrentWindowHandle;
            Thread.Sleep(TimeSpan.FromSeconds(5));

            var allWindowHandles = session.WindowHandles;
            session.SwitchTo().Window(allWindowHandles[0]);
        }

I wish to get this working, because the application i will ultimately be testing can take anything from 1-2 minutes to launch and contains a splash screen.

Any suggestions are much appreciated.

**Comment by anunay1** [comment]
Yes the code that you shared will always launch a new application. Because you are creating a session with notepad not root.

**Comment by merojo** [comment]
I just started reading this project's TODO , any how I noticed the sample for an existing window uses a different param, not "app" , ""appTopLevelWindow" .  
NOT: SetCapability("app"
instead SetCapability("appTopLevelWindow"     
I haven't tried this, just an noticed from RTFM on the link that you posted,
